File: cielo24_Java/cielo24/src/cielo24/json/ElementList.java
package cielo24.json;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class ElementList extends JsonBase
{
	@SerializedName("version")
    public int version;
	@SerializedName("start_time")
    public int startTime;      // Milliseconds
	@SerializedName("end_time")
    public int endTime;        // Milliseconds
	@SerializedName("language")
    public String language;
	@SerializedName("segments")
    public List<Segment> segments;
	@SerializedName("speakers")
    public List<Speaker> speakers;
}

File: cielo24_.NET/Cielo24/JSON/ElementList/ElementList.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Cielo24.JSON.ElementList
{
    public class ElementList : JsonBase
    {
        [JsonProperty("version")]
        public int Version { get; set; }
        [JsonProperty("start_time")]
        public int StartTime { get; set; }      // Milliseconds
        [JsonProperty("end_time")]
        public int EndTime { get; set; }        // Milliseconds
        [JsonProperty("language")]
        public string Language { get; set; }
        [JsonProperty("segments")]
        public List<Segment> Segments { get; set; }
        [JsonProperty("speakers")]
        public List<Speaker> Speakers { get; set; }
    }
}